pkg/cmd/flags/policy.go

```diff
@@ -81,13 +81,13 @@ func PrepareFilterMapsFromPolicies(policies []k8s.PolicyInterface) (PolicyScopeM
 					continue
 				}
 
-				// at this point we know the filter is an event context filter
-				// context filters are provided without "context." prefix so we need to add it
-				evtContextFlags, err := parseEventFlag(fmt.Sprintf("%s.context.%s", r.Event, f))
+				// at this point we know the filter is an event scope filter
+				// scope filters are provided without "scope." prefix so we need to add it
+				evtScopeFlags, err := parseEventFlag(fmt.Sprintf("%s.scope.%s", r.Event, f))
```

Comment:
I think we should name it `evtScopeOpts` differentiating it from the `--scope` flags naming.

Reply:
I think having evt (enent) in the name already differentiates it from the "global" policy scope, does't it?

Reply:
You're right. I concur.